src/main/java/QueryServlet.java

```diff
@@ -105,12 +105,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 + "&key=ea65020114ffc1e71e760341a0285f99e73eabbc");
 
     String fetchUrlString = fetchUrl.toString();
-    String censusTableLink = 
+    String censusTableLink =
         "https://data.census.gov/cedsci/table?tid=ACS"
-            + tableNameToAbbrev.get(fetchUrlString.substring(
-              fetchUrlString.indexOf("acs1/") + 5, fetchUrlString.indexOf("?")))
-            + "1Y" + year + "."
-            + fetchUrlString.substring(fetchUrlString.indexOf(",") + 1, fetchUrlString.indexOf("_"));
+            + tableNameToAbbrev.get(
+                fetchUrlString.substring(
+                    fetchUrlString.indexOf("acs1/") + 5, fetchUrlString.indexOf("?")))
```

Comment:
Isn't fetchUrlString.indexOf("acs1/") + 5 just the data table variable? Or no?

Reply:
Oh, I think this is actually code that Kyra wrote for the data table. @kyracq ?

Reply:
Yes, thanks for catching this will change this to indexOf(dataTablePrefix)